lib/HALTarget/IMU.h

```diff
@@ -95,16 +94,22 @@ class IMU : public IIMU
 
     /**
      * 	Configures the sensors with settings optimized for turn sensing.
+     *
+     *  Additionally, only the z-axis of the gyroscope and the x-axis of the accelerometer are enabled.
      */
     void configureForTurnSensing() final;
 
     /**
      * Takes a reading from the accelerometer and makes the measurements available in a.
+     *
+     * Only the Acceleration along the x-axis is read. The measurements along the other axes are disabled and set to 0.

```

Comment:
This comments should also probably be present in the `get` methods, just to make sure the user knows this explicitely